CRM/Contact/Import/Parser/Contact.php

```diff
@@ -427,10 +426,8 @@ public function import($onDuplicate, &$values) {
             }
           }
           else {
-            $errorMessage = $relatedNewContact['error_message'];
-            array_unshift($values, $errorMessage);
-            $this->setImportStatus((int) $values[count($values) - 1], 'ERROR', $errorMessage);
-            return CRM_Import_Parser::ERROR;
+            $this->setImportStatus((int) $values[count($values) - 1], 'ERROR', $relatedNewContact['error_message']);
```

Comment:
These could be `$rowNumber` - I didn't change them all cos I added that later on in the piece - but I've got a bunch of PRs that would need to be rebased if I change it in this PR - so I think it might be a post-cleanup